repo/blob/s3/s3_storage.go

```diff
@@ -219,6 +228,9 @@ func (s *s3Storage) putBlob(ctx context.Context, b blob.ID, data blob.Bytes, opt
 }
 
 func (s *s3Storage) DeleteBlob(ctx context.Context, b blob.ID) error {
+	ctx, cancel := blob.WithRequestTimeout(ctx, s.RequestTimeout)
```

Comment:
Can this be done in a "wrapper" that implements the `blob.Storage` interface?

Reply:
Ok, it's done, I implemented a wrapper called StorageTimeout, can you take a look ?